lib/cache/cache.go

```diff
@@ -993,15 +995,8 @@ func (c *Cache) fetchAndWatch(ctx context.Context, retry utils.Retry, timer *tim
 // relative to our current view of the world.
 //
 // *note*: this function is only sane to call when the cache and event stream are healthy, and
-// cannot run concurrently with event processing. this function injects additional events into
-// the outbound event stream.
+// cannot run concurrently with event processing.
 func (c *Cache) performRelativeNodeExpiry(ctx context.Context) error {
-	if c.target != cacheTargetAuth {
-		// if we are not the auth cache, we are a downstream cache and can rely upon the
-		// upstream auth cache to perform relative expiry and propagate the changes.
-		return nil
-	}
```

Comment:
Does this performRelativeNodeExpiry functions  needs to be triggered even if a cache was created for `ForWindowsDesktop` or `ForDatabases` where the `types.KindNode` kind is not even present in WatchKind ?